client/src/components/Body.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+class Body extends Component {
+  render() {
+    return (
+      <main className="homePage">
+        <h1 className="pageTitle">Class2 Project</h1>
```

Comment:
Hey Clemy :) What do you think about changing this to a h2 tag and add a paragraph
after that to place a text with the project explanation?